src/general/requiredlabels/constraint.tmpl

```diff
@@ -4,7 +4,7 @@ metadata:
   name: k8srequiredlabels
   annotations:
     metadata.gatekeeper.sh/title: "Required Labels"
-    metadata.gatekeeper.sh/version: 1.0.1
+    metadata.gatekeeper.sh/version: 1.0.2
```

Comment:
```suggestion
    metadata.gatekeeper.sh/version: 1.1.0
```
do we want to at least bump the minor version? @maxsmythe @sozercan

Reply:
Makes sense.